### PR TITLE
dooble: add dooble 2020.05.05

### PR DESCRIPTION
**Pull request by kenmays**
japan-8k

Dooble 2020.05.05 - released May 4, 2020
Built Dooble 1.56f (1.x branch) successfully on Haiku hrev54302 x86
Requires QtWebEngine for Dooble 2020.05.05 (2.x branch).

### CODE REVIEW

**Comment by waddlesplash** [comment]
Please delete the older recipe and patch so we can see the diff.

File: www-client/dooble/patches/dooble-2020.05.05.patchset [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 Date: Sat, 27 May 2017 04:59:44 +1000
 Subject: Fix build for haiku
 
-
-diff --git a/dooble.qt5.pro b/dooble.qt5.pro
-index fc800d3..fd01093 100644
---- a/dooble.qt5.pro
-+++ b/dooble.qt5.pro
-@@ -28,12 +28,12 @@ DEFINES         += DOOBLE_LINKED_WITH_LIBSPOTON \
+diff --git a/boot/home/dooble-build/dooble-2020.05.05-orig/1.x/dooble.qt5.pro b/1.x/dooble.qt5.pro
```
Comment (waddlesplash) [inline www-client/dooble/patches/dooble-2020.05.05.patchset:6]: This patchset is completely wrong. How in the world did you generate it?

Comment (kenmays) [reply www-client/dooble/patches/dooble-2020.05.05.patchset:6]: Patch is fine and works fine.

Comment (waddlesplash) [reply www-client/dooble/patches/dooble-2020.05.05.patchset:6]: It has completely incorrect paths and oddly reordered sections, that's not acceptable.

Comment (kenmays) [reply www-client/dooble/patches/dooble-2020.05.05.patchset]: Fixed.

**Comment by threedeyes** [comment]
Is not a 2020.05.05 version. It the same 1.56f.
We need QtWebEngine for 2020.05.05 version.

**Comment by kenmays** [comment]
The new source code '2020.05.05' package contain both Dooble variants: QtWebKit and QtWebEngine.
Easier to retain the code and track its development. Switching to it is just up to the maintainer(s).

**Comment by Begasus** [comment]
Conflicting files atm ...

**Comment by kenmays** [comment]
@begasus - These conflicting files are removed.

**Comment by Begasus** [comment]
@threedeyes https://repology.org/project/dooble/versions any thoughts on how to proceed with this?

**Comment by Begasus** [comment]
From: https://github.com/textbrowser/dooble/releases the versioning looks ok to me

**Pull request closed without merging**